compute/Dockerfile.compute-node

```diff
@@ -925,8 +925,8 @@ ARG PG_VERSION
 RUN case "${PG_VERSION}" in "v17") \
     echo "pg_session_jwt does not yet have a release that supports pg17" && exit 0;; \
     esac && \
-    wget https://github.com/neondatabase/pg_session_jwt/archive/ff0a72440e8ff584dab24b3f9b7c00c56c660b8e.tar.gz -O pg_session_jwt.tar.gz && \
-    echo "1fbb2b5a339263bcf6daa847fad8bccbc0b451cea6a62e6d3bf232b0087f05cb pg_session_jwt.tar.gz" | sha256sum --check && \
+    wget https://github.com/neondatabase/pg_session_jwt/archive/b30d7df196d0cb816365e565a7579f9c78221206.tar.gz -O pg_session_jwt.tar.gz && \
```

Comment:
one more fix was needed, please change commit hash to `5aee2625af38213650e1a07ae038fdc427250ee4`